src/MassTransit.QuartzIntegration/ScheduleMessageConsumer.cs

```diff
@@ -78,20 +78,21 @@ public ScheduleMessageConsumer(IScheduler scheduler)
                 _scheduler.ScheduleJob(jobDetail, trigger);
         }
 
-        ITrigger CreateTrigger(RecurringSchedule message, IJobDetail jobDetail, TriggerKey triggerKey)
+        ITrigger CreateTrigger(RecurringSchedule schedule, IJobDetail jobDetail, TriggerKey triggerKey)
         {
             TimeZoneInfo tz = TimeZoneInfo.Local;
-            if (!string.IsNullOrWhiteSpace(message.TimeZoneId) && message.TimeZoneId != tz.Id)
-                tz = TimeZoneInfo.FindSystemTimeZoneById(message.TimeZoneId);
+            if (!string.IsNullOrWhiteSpace(schedule.TimeZoneId) && schedule.TimeZoneId != tz.Id)
+                tz = TimeZoneInfo.FindSystemTimeZoneById(schedule.TimeZoneId);
 
             TriggerBuilder triggerBuilder = TriggerBuilder.Create()
                 .ForJob(jobDetail)
                 .WithIdentity(triggerKey)
-                .StartAt(message.StartTime)
-                .WithCronSchedule(message.CronExpression, x =>
+                .StartAt(schedule.StartTime)
+                .WithDescription(schedule.Description)

```

Comment:
No. Here quartz code of TriggerBuilder.WithDescription():
```c#
public TriggerBuilder WithDescription(string description)
{
    this.description = description;
    return this;
}
```
Nothing fancy here.